noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -1289,51 +1706,16 @@
                 ]]>
             </warnings>
         </queries>
-    </rule>
-
-    <rule>
-        <title>Document Description count</title>
-        <group>arkivstruktur</group>
-        <queries>
-            <info>
-                <![CDATA[
-                    SELECT r._ref_arkivdel series_system_id, count(*) number_of_document_description
-                    FROM arkivstruktur.dokumentbeskrivelse d
-                    JOIN arkivstruktur.registrering r ON r.systemid = d._ref_registrering
-                    GROUP BY r._ref_arkivdel
-
-                    UNION ALL
-
-                    SELECT 'total', count(*)
-                    FROM arkivstruktur.dokumentbeskrivelse;
-                ]]>
-            </info>
-        </queries>
-    </rule>
-
-    <rule>
-        <title>Document Descriptions without Document Objects</title>
-        <group>arkivstruktur</group>
-        <queries>
-            <info>
-                <![CDATA[
-                    SELECT r._ref_arkivdel series_system_id, COUNT(*) document_descriptions_without_document_objects
-                    FROM arkivstruktur.dokumentbeskrivelse d
-                    JOIN arkivstruktur.registrering r ON r.systemid = d._ref_registrering
-                    WHERE d.systemid NOT IN (
-                        SELECT _ref_dokumentbeskrivelse
-                        FROM arkivstruktur.dokumentobjekt
-                        WHERE _ref_dokumentbeskrivelse IS NOT NULL
-                        GROUP BY _ref_dokumentbeskrivelse
-                    )
-                    GROUP BY r._ref_arkivdel;
-                ]]>
-            </info>
-        </queries>
-    </rule>
+    </test>
 
-    <rule>
+    <test id="AST14">
         <title>Document Description status</title>
+        <description>
+            Tests whether a non-finalized Document Description exists in the archive.
```

Comment:
`Tests whether non-finalized document descriptions exist in the archive. A document description is considered non-finalized if a finalized date or a finalizing party is not specified, or if the document description status is not 'Dokumentet er ferdigstilt'`